File: C/1204.c
#include <stdio.h>

int main(){
    int n;
    scanf("%d", &n);
    if(n%10==1 && n!=11){
    	printf("%dst", n);
	} else if(n%10==2 && n!=12){
		printf("%dnd", n);
	} else if(n%10==3 && n!=13){
		printf("%drd", n);
	} else{
		printf("%dth", n);
	}
	return 0;
}

File: Python/1204.py
n = int(input())

if n%10==1 and n!=11 : print("%dst" % n)
elif n%10==2 and n!=12 : print("%dnd" % n)
elif n%10==3 and n!=13 : print("%drd" % n)
else : print("%dth" % n)